README.md

```diff
@@ -8,7 +8,11 @@ by Tamal K. Dey and Tao Hou, which appears on the 2022 European Symposium on Alg
 
 ## Change Log
 
-### 2022.8.16
+### 2023.10.01
+
+- We are pleased to announce python bindings of Fast Zigzag
```

Comment:
It should be 'FastZigzag' to align with what we have been calling it. Thanks.

Reply:
Fixed it in the new PR that I will commit.